app/javascript/app/components/emission-pathways-model-table/emission-pathways-model-table-selectors.js

```diff
@@ -0,0 +1,34 @@
+import { createSelector } from 'reselect';
+import isEmpty from 'lodash/isEmpty';
+import pick from 'lodash/pick';
+
+const getCategoryName = data => data.category;
+const getId = data => data.id || null;
+const getData = espData =>
+  (espData.espModelsData && !isEmpty(espData.espModelsData.data)
```

Comment:
can't you send in espModel.data from the container mapStateToProps instead of doing all this checking. Then all you need is an isEmpty?